conda_smithy/configure_feedstock.py

```diff
@@ -1404,15 +1404,15 @@ def commit_changes(forge_file_directory, commit, cs_ver, cfp_ver, cb_ver):
                 if commit == "edit":
                     git_args += ["--edit", "--status", "--verbose"]
                 subprocess.check_call(git_args, cwd=forge_file_directory)
-                print("")
+                logger.info("")
             else:
-                print(
+                logger.info(
```

Comment:
@ihnorton, this is logged as INFO and the default logging level is INFO

Reply:
The problem seems to be that there is no handler configured, so it defaults to the root logger which has a fallback level of WARN. https://github.com/conda-forge/conda-smithy/pull/1181